sympy/ntheory/factor_.py

```diff
@@ -1304,6 +1304,90 @@ def divisor_count(n, modulus=1):
     return Mul(*[v + 1 for k, v in factorint(n).items() if k > 1])
 
 
+def _udivisors(n):
+    """Helper function for udivisors which generates the unitary divisors."""
+
+    factorpows = [p**e for p, e in factorint(n).items()]
+    for i in xrange(2**len(factorpows)):
+        d, j = 1, i
+        for k in xrange(i.bit_length()):
```

Comment:
`bit_length()` method is only valid from ptyhon 2.7, whereas sympy is compatible with 2.6 and less, my two cents